src/app/components/search/search.component.ts

```diff
@@ -108,6 +108,8 @@ export class SearchComponent implements OnInit, OnDestroy {
           ...this.route.snapshot.queryParams,
           text,
           currPage: 1,
+          // Disable any other sorting
+          orderBy: ''
```

Comment:
sorry for not understanding this as clearly, but what would be the purpose of this line here?